typescript/sdk/src/HyperlaneApp.ts

```diff
@@ -45,6 +47,17 @@ export class HyperlaneApp<
     return this.get(chain);
   }
 
+  getFlattenedFilteredContracts(
```

Comment:
Is 'flattened' a term used in solidity to get contract impl? If not I'd suggest `getContractImplementations` and make `filter` an optional param.

Reply:
So this isn't actually getting the implementations, it's flattening out the ProxiedContracts into Contracts

I can't say I love this name either, but given our plans to remove ProxiedContract, it's not long for this world anyhow.